packages/ts-autocomplete/src/index.ts

```diff
@@ -10,20 +11,52 @@ type TypeFilename = string;
 
 type UpdateDefinitionFunction = (newDef: Record<TypeFilename, string>) => void;
 
+// TODO
+/*
+function createIOError(code: string, details = ""): NodeJS.ErrnoException {
+    const err: NodeJS.ErrnoException = new Error(`${code} ${details}`);
+    err.code = code;
+    if (Error.captureStackTrace) Error.captureStackTrace(err, createIOError);
+    return err;
+}
+    */
+function relativeNodePath(fileName: string): string {
```

Comment:
This whole function is kinda the crux of this. If we can't reliably map what ts is trying to read to something we have in our fixture of types both when testing and when doing npm run start in mongosh AND in the bundled mongosh, then none of this will work.

Reply:
To test this I copied these packages to mongosh and built a mongosh binary there. It seemed to work. For a full test we'll have to do a release of this then bump it in mongosh.